public/test.html

```diff
@@ -167,9 +171,7 @@ <h3>Google url scheme</h3>
   <h3>OpenStreetMap url scheme</h3>
   <ol>
     <li><a href="https://www.openstreetmap.org/#map=16/33.89041/35.50664">https://www.openstreetmap.org/#map=16/33.89041/35.50664</a></li>
-    <li><a href="https://openstreetmap.org/#map=16/33.89041/35.50664">https://openstreetmap.org/#map=16/33.89041/35.506641</a></li>
```

Comment:
Why did you remove this and another test link below? They are useful to test.

Reply:
openstreetmap.org without www is not caught anymore https://github.com/organicmaps/organicmaps/pull/4415